blob/azureblob/example_test.go

```diff
@@ -93,14 +91,21 @@ func Example_sasToken() {
 }
 
 func Example_openBucket() {
+	// This example is used in https://gocloud.dev/howto/blob/open-bucket/#azure
+
+	// import _ "gocloud.dev/blob/azureblob"
```

Comment:
Why is this needed?

Reply:
The embedded examples will also show used imports, but it won't show imports not referenced in the example. This is one of the lines gatherexamples processes to add it into the imports shown in the example.

It's useful to show this in documentation because without this blank import, the `blob.OpenBucket` line will return an error (the URL opener won't be linked in).